examples/xyz.js

```diff
@@ -1,41 +1,15 @@
-goog.require('ol.Attribution');
 goog.require('ol.Map');
 goog.require('ol.View');
-goog.require('ol.control');
 goog.require('ol.layer.Tile');
-goog.require('ol.source.OSM');
 goog.require('ol.source.XYZ');
 
 
-var attribution = new ol.Attribution({
-  html: 'Tiles &copy; <a href="http://maps.nls.uk/townplans/glasgow_1.html">' +
-      'National Library of Scotland</a>'
-});
-
```

Comment:
I also [removed the source](https://github.com/openlayers/ol3/pull/4420/files#r45166546) for which this attribution applies.

The problem with this example (and many others) is that people keep tacking additional functionality in without any good documentation.  This is our canonical XYZ example.  People shouldn't have to get puzzled by the weird `/** @type {olx.control.AttributionOptions} */` comment, the unexplainable `attributions` array (why an array with no apparent way to control which gets displayed when).

In addition, this example is ugly and generates a slew of 404s.

![image](https://cloud.githubusercontent.com/assets/41094/11247351/f7809c48-8dd8-11e5-82b5-fcfd8bae87a6.png)

I'd like us to be more considerate of our users in examples (not just dump mysterious code without explanation).  This example isn't a particularly egregious violation of that, but we've got a long way to go to provide simple examples with good documentation, and I hope we don't have to debate every change headed in that direction.